src/test/ov_utils_test.cpp

```diff
@@ -25,83 +25,81 @@ using testing::ElementsAre;
 
 TEST(OVUtils, CopyBlob) {
     const std::vector<size_t> shape{2, 3, 4, 5};
-    const InferenceEngine::Precision precision{InferenceEngine::Precision::FP32};
-    const InferenceEngine::Layout layout{InferenceEngine::Layout::NCHW};
+    const auto elementType = ov::element::Type_t::f32;
     const size_t elementsCount = std::accumulate(shape.begin(), shape.end(), 1, std::multiplies<size_t>());
-    const size_t totalByteSize = elementsCount * precision.size();
-
-    const InferenceEngine::TensorDesc desc{precision, shape, layout};
+    const size_t f32_size = 4;
```

Comment:
This does not need to be hardcoded. Use ov::element::Type::f32.size

Reply:
Few lines down there is the same usage to be exchanged.